framework/src/transfers/MultiAppPostprocessorTransfer.C

```diff
@@ -38,6 +38,12 @@ MultiAppPostprocessorTransfer::validParams()
                              reduction_type,
                              "The type of reduction to perform to reduce postprocessor "
                              "values from multiple SubApps to a single value");
+  params.addParam<bool>(
+      "warn_source_object_execution_schedule",
+      true,
+      "Emit a warning when the transfer execution schedule is detected to lag "
+      "information from the postprocessor. Note that the check cannot detect all "
+      "potential wrong combinations of postprocessor/transfer execution schedules");
```

Comment:
I wonder if you could make a static method on the `MultiAppTransfer` that takes a reference to `InputParameters` and adds this parameter and call it from these derived classes. Then if you ever want to change the doc string or the default you could just do it one place

Reply:
I think I can do it

Reply:
done